pkg/tests/accounts_test.go

```diff
@@ -51,68 +47,73 @@ func Test_TransitionAccount(t *testing.T) {
+	}, ts.Nodes()...)
+	// default wallet block issuer account.
+	blockIssuerAccountOutput := ts.AccountOutput("Genesis:3")
+	ts.AssertAccountData(&accounts.AccountData{
+		ID:              wallet.BlockIssuer.AccountID,
+		Credits:         accounts.NewBlockIssuanceCredits(iotago.MaxBlockIssuanceCredits/2, 0),
+		OutputID:        blockIssuerAccountOutput.OutputID(),
+		ExpirySlot:      iotago.MaxSlotIndex,
+		BlockIssuerKeys: wallet.BlockIssuer.BlockIssuerKeys(),
+	}, ts.Nodes()...)
 
-	// MODIFY EXISTING GENESIS ACCOUNT AND PREPARE SOME BASIC OUTPUTS
-
+	// MODIFY EXISTING GENESIS ACCOUNT
 	newGenesisOutputKey := utils.RandBlockIssuerKey()
-	newAccountBlockIssuerKey := utils.RandBlockIssuerKey()
-	latestCommitmentID := node1.Protocol.MainEngineInstance().Storage.Settings().LatestCommitment().Commitment().MustID()
+	var block1Slot iotago.SlotIndex = 1
+	// set the expiry of the genesis account to be the block slot + max committable age.
+	newExpirySlot := block1Slot + ts.API.ProtocolParameters().MaxCommittableAge()
 
-	accountInput, accountOutputs, accountWallets := ts.TransactionFramework.TransitionAccount(
+	tx1 := ts.DefaultWallet().TransitionAccount(
+		"TX1",
 		"Genesis:1",
-		testsuite.WithAddBlockIssuerKey(newGenesisOutputKey),
-		testsuite.WithBlockIssuerExpirySlot(1),
+		mock.WithAddBlockIssuerKey(newGenesisOutputKey),
+		mock.WithBlockIssuerExpirySlot(newExpirySlot),
 	)
-	consumedInputs, equalOutputs, equalWallets := ts.TransactionFramework.CreateBasicOutputsEqually(4, "Genesis:0")
-
-	tx1 := lo.PanicOnErr(ts.TransactionFramework.CreateSignedTransactionWithOptions("TX1", append(accountWallets, equalWallets...),
-		testsuite.WithAccountInput(accountInput, true),
-		testsuite.WithInputs(consumedInputs),
-		testsuite.WithContextInputs(iotago.TxEssenceContextInputs{
-			&iotago.BlockIssuanceCreditInput{
-				AccountID: genesisAccountOutput.AccountID,
-			},
-			&iotago.CommitmentInput{
-				CommitmentID: latestCommitmentID,
-			},
-		}),
-		testsuite.WithOutputs(append(accountOutputs, equalOutputs...)),
-		testsuite.WithAllotments(iotago.Allotments{&iotago.Allotment{
-			AccountID: genesisAccountOutput.AccountID,
-			Value:     0,
-		}}),
-	))
 
-	var block1Slot iotago.SlotIndex = 1
+	// defaults block issuer issues a block containing the transaction in slot 1.
 	genesisCommitment := iotago.NewEmptyCommitment(ts.API.ProtocolParameters().Version())
 	genesisCommitment.ReferenceManaCost = ts.API.ProtocolParameters().CongestionControlParameters().MinReferenceManaCost
-
-	block1 := ts.IssueBasicBlockAtSlotWithOptions("block1", block1Slot, genesisCommitment, blockIssuer, node1, tx1)
-
+	block1 := ts.IssueBasicBlockAtSlotWithOptions("block1", block1Slot, ts.DefaultWallet(), tx1, mock.WithSlotCommitment(genesisCommitment))
 	latestParent := ts.CommitUntilSlot(ts.BlockID("block1").Slot(), block1)
 
+	// assert diff of a genesis account, it should have a new output ID, same expiry slot and a new block issuer key.
```

Comment:
```suggestion
	// assert diff of the genesis account, it should have a new output ID, same expiry slot and a new block issuer key.
```

There's only one "genesis account" iiuc.